### ISSUE TITLE
Division Sigil not crafting correct ingot

### ISSUE THREAD

**Issue by Keridos** [rwtema/extrautilities_old_issues#782]
The stabilized sigil (after the end ritual) does not craft the mobius ingots for me on forge 10.13.4.1558 and ExU 1.2.12. Does not work with DragonAPI and TConstruct installed.

**Comment by Keridos** [comment]
Should be fixable if you add your recipes in the init phase, not preinit. Not sure if you do not already do that, but Reika mentioned that he does his method that breaks the recipes in preinit.

**Comment by MJRLegends** [comment]
Is this ever going to be fixed as im getting the problem in my mod pack https://github.com/MJRLegends/Space-Astronomy-Feedback-/issues/238

**Comment by Keridos** [comment]
Use the Minetweaker recipe fix from here: https://gist.github.com/Keridos/0fca4ad70638f2542988

**Comment by MJRLegends** [comment]
Yeah i downgraded to Extra Utilities v1.2.10 and it fixed it anyways but i might just use that minetweaker way since i already have that mod in my mod pack umm...

Edit: i think i will use the downgrading method for the minute and do some more testing with changing the recipes via minetweaker because i want to be 100% sure its fixed using that method. Thanks a lot for the link :P
